CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.3.1
```

Comment:
how come changelog is only 1.3.1?  in their tap i see it at 1.3.5

https://github.com/singer-io/tap-s3-csv/tags